Doppler.Sap/Services/BillingServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using Doppler.Sap.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Doppler.Sap.Services
+{
+    public static class ServicesCollectionExtensions
+    {
+        public static void AddServices(this IServiceCollection services, IConfiguration configuration)
```

Comment:
Is it possible to use a more descriptive name?

Reply:
For the class or to the method or both?

Reply:
The method, for example:

```suggestion
        public static void AddSapServices(this IServiceCollection services, IConfiguration configuration)
```